README.md

```diff
@@ -8,12 +8,13 @@ https://learn.microsoft.com/en-us/azure/devops/extend/get-started/node?view=azur
 
 ## Testing changes on local environment 
```

Comment:
This says 'Testing changes on local environment'. Should it be 'Release new version'?

Reply:
However, not fully sure as it says '<Your-publisher-name>'

Reply:
Modified